pywhat/Data/regex.json

```diff
@@ -1309,6 +1309,23 @@
          ]
       }
    },
+   {
+      "Name": "UPS Tracking Number",
+      "Regex": "^(1Z[0-9A-Z]{6}[0-9]{2}[0-9]{8})$",
+      "plural_name": false,
+      "Description": null,
+      "Rarity": 1,
+      "URL": "https://www.ups.com/track?tracknum=",
+      "Tags": [
+         "Tracking"
+      ],
```

Comment:
We have some tags, like `Identifiers` and `Bug Bounty` that could fit here. Maybe some others.